rfcs/0000-tsdb-dimensions.md

```diff
@@ -0,0 +1,332 @@
+    service of the request.
+  reusable:
+    top_level: true
+    expected:
+      - at: service
+        as: origin
+        beta: Reusing the `service` fields in this location is currently considered beta.
+        short_override: Describes the origin service in case of an incoming request or event.
+      - at: service
+        as: target
+        beta: Reusing the `service` fields in this location is currently considered beta.
+        short_override: Describes the target service in case of an outgoing request or event.
+  type: group
+  fields:
+
+    - name: address
+      level: extended
+      type: keyword
+      dimension: true
+      short: Address of this service.
+      description: >
+        Address where data about this service was collected from.
+
+        This should be a URI, network address (ipv4:port or [ipv6]:port) or a resource path (sockets).
+      example: 172.26.0.2:5432
+
+```
+Changes to host mapping
+
+```yaml
+---
+- name: host
+  title: Host
+  group: 2
+  short: Fields describing the relevant computing instance.
+  description: >
+    A host is defined as a general computing instance.
+
+    ECS host.* fields should be populated with details about the host on which
+    the event happened, or from which the measurement was taken.
+    Host types include hardware, virtual machines, Docker containers, and Kubernetes nodes.
+  type: group
+  fields:
+    - name: name
+      level: core
+      type: keyword
+      short: Name of the host.
+      dimension: true
+      description: >
+        Name of the host.
+        It can contain what hostname returns on Unix systems, the fully
+        qualified domain name (FQDN), or a name specified by the user.
+        The recommended value is the lowercase FQDN of the host.
+
+```
+Changes to agent mapping
+
+```yaml
+---
+- name: agent
+  title: Agent
+  group: 2
+  short: Fields about the monitoring agent.
+  description: >
+    The agent fields contain the data about the software entity, if any, that collects, detects, or observes events on a host, or takes measurements on a host.
+
+    Examples include Beats. Agents may also run on observers. ECS agent.* fields shall be populated with details of the agent running on the host or observer where the event happened or the measurement was taken.
+  footnote: >
+    Examples: In the case of Beats for logs, the agent.name is filebeat. For APM, it is the
+    agent running in the app/service. The agent information does not change if
+    data is sent through queuing systems like Kafka, Redis, or processing systems
+    such as Logstash or APM Server.
+  type: group
+  fields:
+
+    - name: id
+      level: core
+      type: keyword
+      short: Unique identifier of this agent.
+      description: >
+        Unique identifier of this agent (if one exists).
+
+        Example: For Beats this would be beat.id.
+      example: 8a4f500d
+      dimension: true
```

Comment:
Nit: Can we have a consistent ordering where dimension fields are added in this document. Some times it is added before and some time it is after the description field.